src/isc-dhcp/patch/0007-If-destination-of-BOOTREQUEST-is-directed-broadcast-.patch

```diff
@@ -12,52 +12,44 @@ Subject: [PATCH 3/3] If destination of BOOTREQUEST is directed broadcast,
  3 files changed, 131 insertions(+), 16 deletions(-)
 
 diff --git a/common/discover.c b/common/discover.c
-index 8e7f632..73eb8a9 100644
+index 98ac46a..53afecc 100644
 --- a/common/discover.c
 +++ b/common/discover.c
-@@ -227,6 +227,7 @@ struct iface_conf_list {
+@@ -236,6 +236,7 @@ struct iface_conf_list {
  struct iface_info {
  	char name[IF_NAMESIZE+1];	/* name of the interface, e.g. "bge0" */
  	struct sockaddr_storage addr;	/* address information */
-+    struct sockaddr_storage netmask;    /* netmask information */
++	struct sockaddr_storage netmask;    /* netmask information */
  	isc_uint64_t flags;		/* interface flags, e.g. IFF_LOOPBACK */
  };
  
-@@ -401,6 +402,7 @@ struct iface_conf_list {
- struct iface_info {
- 	char name[IFNAMSIZ];		/* name of the interface, e.g. "eth0" */
- 	struct sockaddr_storage addr;	/* address information */
-+    struct sockaddr_storage netmask;    /* netmask information */
- 	isc_uint64_t flags;		/* interface flags, e.g. IFF_LOOPBACK */
- };
- 
-@@ -576,6 +578,17 @@ next_iface4(struct iface_info *info, int *err, struct iface_conf_list *ifaces) {
- 		}
- 		memcpy(&info->addr, &tmp.ifr_addr, sizeof(tmp.ifr_addr));
+@@ -367,6 +368,17 @@ next_iface(struct iface_info *info, int *err, struct iface_conf_list *ifaces) {
+ 	}
+ 	info->flags = tmp.lifr_flags;
  
-+		if (ioctl(ifaces->sock, SIOCGIFNETMASK, &tmp) < 0) {
-+			if (errno == EADDRNOTAVAIL) {
-+				continue;
-+			}
-+			log_error("Error getting netmask "
-+				  "for '%s'; %m", name);
-+			*err = 1;
-+			return 0;
-+		}
-+		memcpy(&info->netmask, &tmp.ifr_netmask, sizeof(tmp.ifr_netmask));
-+
- 		memset(&tmp, 0, sizeof(tmp));
- 		strncpy(tmp.ifr_name, name, sizeof(tmp.ifr_name) - 1);
- 		if (ioctl(ifaces->sock, SIOCGIFFLAGS, &tmp) < 0) {
-@@ -780,6 +793,7 @@ struct iface_conf_list {
++	 if (ioctl(ifaces->sock, SIOCGIFNETMASK, &tmp) < 0) {
++		 if (errno == EADDRNOTAVAIL) {
++		 continue;
++		 }
++		 log_error("Error getting netmask "
++			 "for '%s'; %m", name);
++		 *err = 1;
++		 return 0;
++	 }
++	 memcpy(&info->netmask, &tmp.ifr_netmask, sizeof(tmp.ifr_netmask));
++
+ 	ifaces->next++;
+ 	*err = 0;
+ 	return 1;
+@@ -410,6 +422,7 @@ struct iface_conf_list {
  struct iface_info {
  	char name[IFNAMSIZ];		/* name of the interface, e.g. "bge0" */
  	struct sockaddr_storage addr;	/* address information */
-+    struct sockaddr_storage netmask;    /* netmask information */
++	struct sockaddr_storage netmask;	 /* netmask information */
```

Comment:
The leading tab is now correct, but the tab preceding the comment is followed by a single space. Please delete that space.